src/pages/HomePage.tsx

```diff
@@ -42,47 +39,32 @@ export const HomePage: React.FC = () => {
     }
   };
 
-  const toggleDialog = () => {
-    setIsDialogOpen(!isDialogOpen);
-  };
-
-  const resetSelectedMovie = () => {
-    setSelectedMovie(null);
-  };
-
   const handleMovieClick = (movie: MovieInfo) => {
     navigate(`movies/${movie.id}?search=${searchQuery}&genre=${selectedGenre}&sortBy=${sortBy}`);
```

Comment:
Just FYI:
you can use the navigate method like this:
![image](https://github.com/boryskrutskykh/react-movie-app/assets/40394400/1c470a5a-d7a8-457d-bf52-59f092919460)

and no need to put all the query params manually